README.md

```diff
@@ -1,57 +1,47 @@
-# Project Name
+# Responsive web design- Clone of Thenextweb.com
 
-> One paragraph statement about the project.
+> This project consists of replicating the The Next Web website putting emphasis on how it behaves differently depending on the size of the screen where the website is rendered.
 
-![screenshot](./app_screenshot.png)
+![screenshot](./images/tnw-mobileview.png)
+![screenshot](./images/tnw-tabletview.png)
+![screenshot](./images/tnw-desktopview.png)
 
```

Comment:
- [ ] Great template and Readme description!!

Reply:
Thanks